service/tcp.go

```diff
@@ -124,29 +122,25 @@ type tcpService struct {
 
 // NewTCPService creates a TCPService
 // `replayCache` is a pointer to SSServer.replayCache, to share the cache among all ports.
-func NewTCPService(ciphers CipherList, replayCache *ReplayCache, m metrics.ShadowsocksMetrics, timeout time.Duration) TCPService {
-	return &tcpService{
+func NewTCPHandler(port int, ciphers CipherList, replayCache *ReplayCache, m metrics.ShadowsocksMetrics, timeout time.Duration) TCPHandler {
+	return (&tcpHandler{
+		port:              port,
 		ciphers:           ciphers,
 		m:                 m,
 		readTimeout:       timeout,
 		replayCache:       replayCache,
 		targetIPValidator: onet.RequirePublicIP,
-	}
+	})
```

Comment:
*nit* why do we wrap the tcpHandler in brackets `()` here?

Reply:
Removed.